### PR TITLE
Add FlowPatternModel enum

### PR DESCRIPTION
**Pull request by arthursoprana**

### CODE REVIEW

**Comment by oliveira-mauricio** [comment]
This new property is not being exported/imported to .alfacase, is that intended?

File: src/alfasim_sdk/__init__.py [hunk 1]
```diff
@@ -255,6 +255,7 @@ def get_alfasim_sdk_api_path():
 from alfasim_sdk._internal.constants import PipeThermalModelType
 from alfasim_sdk._internal.constants import PipeThermalPositionInput
 from alfasim_sdk._internal.constants import PumpType
+from alfasim_sdk._internal.constants import PVTCompositionalViscosityModel
```
Comment (prusse-martin) [inline src/alfasim_sdk/__init__.py:258]: Do you have any idea of how those did sneak in in the wrong order?
Don't we run precommit on ci?

**Review by prusse-martin — APPROVED** [review]

File: src/alfasim_sdk/__init__.py (hunk 1 above)
Comment (arthursoprana) [reply src/alfasim_sdk/__init__.py:258]: Not sure, but I've sorted them in this PR

**Comment by arthursoprana** [comment]
> This new property is not being exported/imported to .alfacase, is that intentional?
> 
> @arthursoprana just to make sure you read this

@oliveira-mauricio I'll add it as a TODO task of ASIM-4574

**Review by oliveira-mauricio — APPROVED** [review]

**Pull request merged**